MUXSDKImaListener/Classes/include/module.modulemap

```diff
@@ -0,0 +1,3 @@
+module Mux_Stats_Google_IMA {
```

Comment:
I had to move the header into the include directpry as per https://github.com/apple/swift-package-manager/blob/main/Documentation/Usage.md#creating-c-language-targets

Even though that should have created the modulemap automatically, it didn't...so I just created it myself. This allowed the DemoApp to pull in the IMA package, all related dependencies and successfully build